apps/schools/domains/organization/redux/interfaces.ts

```diff
@@ -0,0 +1,239 @@
+import { GetEmployee } from '../../employee/redux/interfaces'
+import { GetPhoto } from '../../common/redux/interfaces'
+
+export interface GetOrganizationSender {
+    id?: string,
+    name?: string,
+    inn?: string,
+}
+
+export interface AllOrganizationsData {
+    id?: string,
+    name?: string,
+    inn?: string,
+    ids?: string,
+    'or_search'?: string,
+    page?: number,
+    'page_size'?: number,
+}
+
+export interface CreateOrganizationResponse {
+    'creator_employee': GetEmployee,
+}
```

Comment:
can we use special function that generates Response interfaces?
it is not convinient to write Response for each request